packages/flutter_tools/lib/src/base/pub_spec_content.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:yaml/yaml.dart';
+
+class PubContent {
+  PubContent(this.content);
+  final YamlMap content;
+
+  String? get appName {
+    return content['name'] as String?;
+  }
+
+  YamlMap get flutterNode {
+    return content['flutter'] as YamlMap;
+  }
+
+  bool isFlutterPackage() {
```

Comment:
style nit: this can be a getter, since it doesn't accept any parameters
```suggestion
  bool get isFlutterPackage {
```